src/main/java/org/jenkinsci/plugins/p4/scm/SwarmScmSource.java

```diff
@@ -2,8 +2,11 @@
 
 import edu.umd.cs.findbugs.annotations.NonNull;
 import hudson.Extension;
+import hudson.model.Action;
 import hudson.model.TaskListener;
-import jenkins.scm.api.SCMHeadCategory;
+import jenkins.scm.api.*;
```

Comment:
Please avoid wildcard imports (for non-static)

Reply:
updated to remove the wildcard